documentation/content/guidebook/drizzle-orm/index.md

```diff
@@ -255,44 +255,44 @@ Make sure to change the table names accordingly.
 
 ```ts
 // schema.js
-import { pgTable, bigint, varchar, boolean } from "drizzle-orm/pg-core";
+import { sqliteTable, text, blob } from "drizzle-orm/sqlite-core";
 
-export const user = pgTable("user", {
-	id: varchar("id", {
-		length: 15 // change this when using custom user ids
-	}).primaryKey()
-	// other user attributes
+export const user = sqliteTable("user", {
+  id: text("id", {
+    length: 15, // change this when using custom user ids
```

Comment:
I don't think we need to define length here since it's not required in SQLite

Reply:
updated at [Commit](https://github.com/pilcrowOnPaper/lucia/pull/934/commits/8738cd1ca90c739fa852e5362269ffe806e54d5a)